### DISCUSSION TITLE
Adding Fatture in Cloud C# SDK to a VB.NET project

### DISCUSSION THREAD

**Discussion by ghost** [fattureincloud/api#26] [Show and tell]
Hi

As per @valmoz 's request I'm going to share a basic tutorial on how to add the Fatture in Cloud SDK for C# to a VB.NET project

1. Open a VB.NET project
2. Open the drop down menu "**Tools**" -> "**NuGet Package Manager**" -> "**Manage NuGet Packages for solution**"
3. On the NuGet Manager screen, click on "**browse**" tab and search for "Fatture in Cloud" or just "fatture". Our package is named "**It.FattureInCloud.Sdk**", select the package and click "**Install**"
4. (OPTIONAL) In case you're missing required dependencies (additional packages/libs required by Fatture in Cloud SDK) you'll get prompted about and install them
5. Once you're done installing the package you'll be able to use the Fatture in Cloud SDK's classes/functions
6. (OPTIONAL/RECOMMENDED) Read Fatture in Cloud's docs about both APIs and SDK functions, the laters are like "wrappers" around APIs, docs explain the parameters/required parameters and response for each method.
7. Code sample (GET companies list): [https://pastebin.com/WLfwby0k](url)

That's a very basic tutorial, also the first one I make :)
I hope it can be helpfull to you!

Thanks
Regards

**Comment by valmoz** [comment]
Thank you @diegonogaretti
